docs/development/bom.md

```diff
@@ -5,49 +5,54 @@ help us manage our software supply chain security by being aware of the open
+| [Kubernetes Python Client](https://pypi.org/project/kubernetes/)           | 26.1.0      | 26.1.0         | managing Kubernetes |
+| [prometheus-client](https://pypi.org/project/prometheus-client/)           | 0.16.0      | 0.16.0         | metrics exposure |
+| [werkzeug](https://pypi.org/project/Werkzeug/)                             | 2.2.3       | 2.2.3          | serving SocketIO server |
+| [GitPython](https://pypi.org/project/GitPython/)                           | 3.1.31      | 3.1.31         | cloning job repositories |
+| [Python markdown](https://pypi.org/project/Markdown/)                      | 3.4.1       | 3.4.1          | rendering docs in Dashboard |
+| [FastAPI](https://github.com/tiangolo/fastapi)                             | 0.92.0      | 0.92.0         | serving API by Python components |
+| [kubectl](https://github.com/kubernetes/kubectl)                           | 1.26.2      | 1.26.2         | managing Kubernetes |
+| [psycopg2-binary](https://pypi.org/project/psycopg2-binary/)               | 2.9.5       | 2.9.5          | connecting to PostgreSQL |
+| [a2wsgi](https://pypi.org/project/a2wsgi/)                                 | 1.7.0       | 1.7.0          | serving Django app in ASGI server |
+| [PyYAML](https://pypi.org/project/PyYAML/)                                 | 6.0         | 6.0            | parsing YAML config files |
+| [Jinja2](https://pypi.org/project/Jinja2/)                                 | 3.1.2       | 3.1.2          | templating Dockerfiles |
+| [schedule](https://pypi.org/project/schedule/)                             | 1.1.0       | 1.1.0          | running periodic tasks |
+| [psutil](https://pypi.org/project/psutil/)                                 | 5.9.4       | 5.9.4          | monitoring running processes |
+| [pydantic](https://pypi.org/project/pydantic/)                             | 1.10.5      | 1.10.5         | schema validation |
+| [typer](https://pypi.org/project/typer/)                                   | 0.7.0       | 0.7.0          | parsing command line arguments |
+| [uvicorn](https://pypi.org/project/uvicorn/)                               | 0.20.0      | 0.20.0         | serving ASGI app |
+| [httpx](https://pypi.org/project/httpx/)                                   | 0.23.3      | 0.23.3         | making HTTP requests |
+| [pyjwt](https://pypi.org/project/PyJWT/)                                   | 2.6.0       | 2.6.0          | handling JWT tokens |
+| [python-multipart](https://pypi.org/project/python-multipart/)             | 0.0.6       | 0.0.6          | handling uploaded plugins |
+| [watchdog](https://pypi.org/project/watchdog/)                             | 2.3.1       | 2.3.1          | watching for changes in plugins |
+| [protobuf](https://pypi.org/project/protobuf/)                             | 4.22.0      | 4.22.0         | handling OTLP protocol |
```

Comment:
```suggestion
| [protobuf](https://pypi.org/project/protobuf/)                             | 4.22.0      | 4.22.0         | handling OTLP |
```

OTLP stands for OpenTelemetry Line Protocol. No need to have a "protocol" there, unless you want to spell out the abbreviation in it's entirety.